scripts/process_twitter_data.py

```
@@ -0,0 +1,260 @@
+            "data": data,
+            "auto_execute": auto_execute,
+            "trust_distance": trust_distance,
+            "pub_key": pub_key
+        }
+        
+        response = requests.post(INGESTION_ENDPOINT, json=payload, timeout=60)
+        
+        if response.status_code == 200:
+            result = response.json()
+            if result.get("success", False):
+                print(f"✅ Successfully processed data")
```

Ruff (0.14.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)